doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -110,39 +114,45 @@ If you experience any problems during the process, restart the Programmer app by
 
       Programmer - Add file
 
-#. Navigate to where you extracted the firmware, and then to the :file:`img_app_bl` folder there.
-#. Select either :file:`nrf9160dk_asset_tracker_v2_ltem_<version-number>.hex` (LTE-M mode) or :file:`nrf9160dk_asset_tracker_v2_nbiot_<version-number>.hex` (NB-IoT mode), depending on where you are located.
-   Check with your SIM card provider for the mode supported at your location.
-   If you are using the iBasis SIM card provided with the DK, you can see `iBasis IoT network coverage`_ .
-   You can use either mode if your location has support for both.
+#. Navigate to where you extracted the firmware, and choose the :file:`mfwnrf9160_<version-number>.zip` file.
+#. Click :guilabel:`Write` in the :guilabel:`Device` section of the side panel.
 
-   For NB-IoT, there is a second variant of the firmware in the :file:`nrf9160dk_asset_tracker_v2_nbiot_legacy_pco_<version-number>.hex` file.
-   Only use this legacy variant if your network does not support ePCO.
+   .. figure:: /images/programmer_write_nrf9160dk.png
+      :alt: Programmer - Write
```

Comment:
The color and the thickness of the highlight box are a bit different from the other screenshots. Could you check?